src/pages/hcp/api-docs/consul/[[...page]].tsx

```diff
@@ -92,11 +93,20 @@ export const getStaticProps: GetStaticProps<
 	const versionData = await fetchCloudApiVersionData(
 		PAGE_CONFIG.githubSourceDirectory
 	)
+	/**
+	 * TODO: add in support for conditionally publishing specs in hcp-specs repo
+	 * For now just manually filtering out this spec as per request in slack.
+	 * https://hashicorp.slack.com/archives/C5FSPUGDS/p1698178472462449
+	 */
+	const filteredVersionData = versionData.filter(
+		(version: ApiDocsVersionData) => version.versionId !== '2022-02-15'
```

Comment:
does this filter out this version or only include it?

it should be this instead right?

```suggestion
		(version: ApiDocsVersionData) => version.versionId !== '2022-02-15'
```

Reply:
I think I pushed up that change at the exact same time you suggested it! 😆 Thank you